test/integration/specs/load-twitter-component.js

```diff
@@ -7,9 +7,11 @@ const mock = simple.mock;
 const ipc = require("node-ipc");
 let localMessagingModule = require('local-messaging-module');
 let testTweets = null;
+const fs = require("fs");
 
 describe('Twitter Component - Integration', () => {
   before(()=>{
+    simple.mock(fs, "watch").callbackWith();
```

Comment:
@rudibravo I think just using mock should do it instead of simple.mock. Thoughts? Cheers

Reply:
@rodrigopavezi sure